frontend/src/exporter/ExportedInsight/ExportedInsight.tsx

```diff
@@ -98,7 +98,7 @@ export function ExportedInsight({
                 >
                     {!!query ? (
                         !!insight.result ? (
-                            <Query query={query} readOnly={true} />
+                            <Query query={query} readOnly={true} cachedResults={insight.result} />
```

Comment:
Completely flyby 🪂 question with very little previous context: Could `query={{ ...query, results: insight.result }}` work as well?

Also, I see `readOnly={true}`. The idea for me was always that if `setQuery` is not present, then it's effectively read only mode 🤔

Reply:
To make my PRing life easier (and since they're not new here)

I'll fix those in follow-up ideally